bin/GameIndex.yaml

```diff
@@ -3845,11 +3854,14 @@ SCPS-19316:
   region: "NTSC-J"
   gameFixes:
     - VU0KickstartHack # Fixes Character SPS.
+    - RatchetDynaHack # Dynamically patches bad COP2 code in Ratchet and Clank games
 SCPS-19317:
   name: "Ratchet & Clank 2 - Going Commando [PlayStation 2 The Best]"
   region: "NTSC-J"
   gameFixes:
     - VU0KickstartHack # Fixes Character SPS.
+    - EETimingHack # Fixes DMA errors 
```

Comment:
This timing hack needs to be removed.

Reply:
Have you played through the Japanese version to make sure it isn't required?